postgres/datadog_checks/postgres/metrics_cache.py

```diff
@@ -141,20 +145,24 @@ def get_archiver_metrics(self, version):
             'relation': False,
         }
 
-    def get_replication_metrics(self, version):
+    def get_replication_metrics(self, version, is_aurora):
         """ Use either REPLICATION_METRICS_10, REPLICATION_METRICS_9_1, or
         REPLICATION_METRICS_9_1 + REPLICATION_METRICS_9_2, depending on the
         postgres version.
-        Uses a dictionnary to save the result for each instance
+        Uses a dictionary to save the result for each instance
         """
         metrics = self.replication_metrics
         if version >= V10 and metrics is None:
             self.replication_metrics = dict(REPLICATION_METRICS_10)
             metrics = self.replication_metrics
         elif version >= V9_1 and metrics is None:
-            self.replication_metrics = dict(REPLICATION_METRICS_9_1)
-            if version >= V9_2:
-                self.replication_metrics.update(REPLICATION_METRICS_9_2)
+            if is_aurora:
```

Comment:
Will v10 also cause the warning/error from `pg_last_xlog_receive_location()`?

https://github.com/DataDog/integrations-core/blob/master/postgres/datadog_checks/postgres/util.py#L217-L220

Reply:
10 is built like this:

```
q1 = (
    'CASE WHEN pg_last_wal_receive_lsn() = pg_last_wal_replay_lsn() THEN 0 ELSE GREATEST '
    '(0, EXTRACT (EPOCH FROM now() - pg_last_xact_replay_timestamp())) END'
)
q2 = 'abs(pg_wal_lsn_diff(pg_last_wal_receive_lsn(), pg_last_wal_replay_lsn()))'
REPLICATION_METRICS_10 = {
    q1: ('postgresql.replication_delay', AgentCheck.gauge),
    q2: ('postgresql.replication_delay_bytes', AgentCheck.gauge),
}
```